src/test/java/com/ksm/wstbackoffice/controller/BaseController.java

```diff
@@ -0,0 +1,19 @@
+package com.ksm.wstbackoffice.controller;
+
+import io.restassured.RestAssured;
+import org.junit.jupiter.api.BeforeEach;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.web.server.LocalServerPort;
+import org.springframework.test.context.ActiveProfiles;
+
+@SpringBootTest(webEnvironment= SpringBootTest.WebEnvironment.RANDOM_PORT)
+@ActiveProfiles("test")
+public class BaseController {
```

Comment:
Please rename to BaseCantrollerIT.
IT stands for Integration Test

Reply:
done